priv/emqx.schema

```diff
@@ -1531,6 +1531,14 @@ end}.
   {datatype, string}
 ]}.
 
+{mapping, "listener.ws.$name.proxy_address_header", "emqx.listeners", [
+  {datatype, {enum, [x_forwarded_for]}}
```

Comment:
should be a string type,
as a user, I would not like to edit schema if I want to use a different header name.

Reply:
Add default value "X-Forwarded-For"